tabs/ports.js

```diff
@@ -4,6 +4,7 @@ TABS.ports = {};
 
 TABS.ports.initialize = function (callback, scrollPosition) {
     var board_definition = {};
+    var isSupportPeripherals = semver.gte(CONFIG.apiVersion, "1.27.0");
```

Comment:
Actual MSP API didn't change. I think it's better to check for firmware version, not for apiVersion. @DzikuVx what do you think?